android/src/main/java/com/reactnativecommunity/cookies/CookieManagerModule.java

```diff
@@ -43,14 +43,21 @@
     private static final String CLEAR_BY_NAME_NOT_SUPPORTED = "Cannot remove a single cookie by name on Android";
     private static final String INVALID_DOMAINS = "Cookie URL host %s and domain %s mismatched. The cookie won't set correctly.";
 
-    private CookieManager mCookieManager;
     private CookieSyncManager mCookieSyncManager;
 
     CookieManagerModule(ReactApplicationContext context) {
         super(context);
         this.mCookieSyncManager = CookieSyncManager.createInstance(context);
-        this.mCookieManager = CookieManager.getInstance();
```

Comment:
Lazy get `CookieManager` instance - basically try to only crash when actually using the library